src/Samples/Sample.AspNetCore.SystemTests/Test/Base/TestBase.cs

```diff
@@ -37,7 +37,7 @@ public void OneTimeSetUp()
         [SetUp]
         public void SetUp()
         {
-            this._testWebApplicationFactory = new TestWebApplicationFactory();
+            _testWebApplicationFactory = new TestWebApplicationFactory();

```

Comment:
Underscore?! Purge it from the codebase! PURGE!

![Purge](https://user-images.githubusercontent.com/12283/99713754-fa766b80-2aa4-11eb-9de6-5a9b03356b56.gif)

```suggestion
            this.testWebApplicationFactory = new TestWebApplicationFactory();
```